page-objects/pages/NavigationLeftPage.js

```diff
@@ -80,114 +78,135 @@ const navItemHelpContact = navItemString2ndLevel.replace('XXX', 'Kontakt');
 const item = '.sidebar-list .link-name';
 const subitem = '.subitem .link-name';
 
+
 async function clickNavItemLogo() {
-	await loadPageFromNavigationItem(navItemLogo);
+	await clickElementFromNavigation(navItemLogo);
 }
 
 async function clickNavItemDashboard() {
-	await loadPageFromNavigationItem(navItemDashboard);
+	await clickElementFromNavigation(navItemDashboard);
 }
 
 async function clickNavItemCourses() {
-	await loadPageFromNavigationItem(navItemCourses);
+	await clickElementFromNavigation(navItemCourses);
 }
 
 async function clickNavItemTasks() {
-	await loadPageFromNavigationItem(navItemTasks);
+	await clickElementFromNavigation(navItemTasks);
 }
 
 async function clickNavItemTasksAsked() {
-	await loadPageFromNavigationItem(navItemTasksAsked);
+	await clickElementFromNavigation(navItemTasksAsked);
 }
 async function clickNavItemTasksPrivate() {
-	await loadPageFromNavigationItem(navItemTasksPrivate);
+	await clickElementFromNavigation(navItemTasksPrivate);
 }
 async function clickNavItemTasksArchive() {
-	await loadPageFromNavigationItem(navItemTasksArchive);
+	await clickElementFromNavigation(navItemTasksArchive);
 }
 
 async function clickNavItemTeams() {
-	await loadPageFromNavigationItem(navItemTeams);
+	await clickElementFromNavigation(navItemTeams);
 }
 
 async function clickNavItemFiles() {
-	await loadPageFromNavigationItem(navItemFiles);
+	await clickElementFromNavigation(navItemFiles);
 }
 
 async function clickNavItemFilesMy() {
-	await loadPageFromNavigationItem(navItemFilesMy);
+	await clickElementFromNavigation(navItemFilesMy);
 }
 
 async function clickNavItemFilesCourses() {
-	await loadPageFromNavigationItem(navItemFilesCourses);
+	await clickElementFromNavigation(navItemFilesCourses);
 }
 
 async function clickNavItemFilesTeams() {
-	await loadPageFromNavigationItem(navItemFilesTeams);
+	await clickElementFromNavigation(navItemFilesTeams);
 }
 
 async function clickNavItemFilesShared() {
-	await loadPageFromNavigationItem(navItemFilesShared);
+	await clickElementFromNavigation(navItemFilesShared);
 }
 
 async function clickNavItemNews() {
-	await loadPageFromNavigationItem(navItemNews);
+	await clickElementFromNavigation(navItemNews);
 }
 
 async function clickNavItemCalendar() {
-	await loadPageFromNavigationItem(navItemCalendar);
+	await clickElementFromNavigation(navItemCalendar);
 }
 
 async function clickNavItemContent() {
-	await loadPageFromNavigationItemNuxtClient(navItemLearnstore);
+
+	await clickElementFromNavigation(navItemLearnstore);
```

Comment:
Used to be await loadPageFromNavigationItemNuxtClient(navItemLearnstore); in main.